AAP-4189 (#365)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/aap-on-azure/proc-azure-nw-private-deploy-ssh-tunnel.adoc

When VPN is not an option and your local users are more technical, the SSH tunnel approach is a secure alternative that enables users to access {PlatformNameShort} from a browser on a local machine.

> In this model, a lightweight linux SSH server is created in a DMZ network, similar to the Azure Hosted Virtual Machine method.

Reviewer: This model creates a lightweight linux SSH server in a DMZ network,

. Update your local hosts file so that the {PlatformNameShort} URLs route traffic to your local machine rather than the private IP that DNS records are configured with.
Add the following line to your hosts file:
+
[subs="+quotes"]
----

> 127.0.0.1 controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com

Reviewer 2: Is "instance" here the right word or would "hostname" suit better?

----
+
The following example shows the line in a hosts file:
+
[subs="+quotes"]
----
##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting. Do not change this entry.
#
127.0.0.1       localhost
255.255.255.255 broadcasthost
::1             localhost

127.0.0.1 controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com

----

> . Run an `ssh` command to establish the SSH tunnel.
> This requires root access on your local machine.

Reviewer 2: ```suggestion
. As a user with root privileges, run the `ssh` command to establish an SSH tunnel.
```

Reviewer 2: You could probably mix it up a little more but I think it makes sense to include the root privileges as part of the step.

"This requires root access on your local machine." - With this construction the pronoun "this" is not clearly referenced in my opinion. Do you mean "this step", "this command", or something else? It's a minor but unreferenced antecedents are a real thing.

I would also `s/an ssh command/the ssh command/`. There is only one ssh command so I think a definite article is more appropriate.

Likewise `s/the SSH tunnel/an SSH tunnel/` because it does not yet exist and using "the" here suggests you are referring to something that already exists.

+
In the example below, the IP address is the address of the SSH server in your DMZ.
Change this IP address for your SSH tunnel.
The `-L` flag makes your local system route traffic for the {ControllerName} URL over port 443 (HTTPs).
+
[subs="+quotes"]
----
sudo ssh azureuser@20.231.60.123 -i ~/.ssh/id_ssh_key -N -f -L 443:controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com:443
----
+
[NOTE]
====

> You must use port 443 must on both sides of the routing path. Using a different port on the local machine causes some {PlatformNameShort} features to not function properly.

Reviewer: You must use port 443 on both sides
'must' repeated

====

> Once the SSH tunnel establishes and your Azure routing is configured, you can access the {ControllerName} URL from your local browser at \https://controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com.

Reviewer: When the SSH tunnel establishes
